pyanaconda/storage/autopart.py

```diff
@@ -36,76 +34,11 @@
 from pykickstart.constants import AUTOPART_TYPE_BTRFS, AUTOPART_TYPE_LVM, AUTOPART_TYPE_LVM_THINP, AUTOPART_TYPE_PLAIN
 
 from pyanaconda.core.i18n import _
-from pyanaconda.storage.utils import get_available_disk_space
+from pyanaconda.storage.utils import get_available_disk_space, suggest_swap_size
```

Comment:
Change `swap_suggestion` in the 1st commit message. It could be name of the function which is `suggest_swap_size` or without `_` character.

Reply:
The same trend follow on some other commits. Is that intentional?